client/js/lib/core/lib_documentation.md

```diff
@@ -1,3 +1,16 @@
 ## Scripts, their effects and where they are used
 
+| CharCount.js | count characters in input fields | public and planning area |
+| CheckableItem.js | toggle checkboxes of multiple items | public and planning area |
 | ExtendedVueEvents.js | adds custom dplan taste to vue events | ? |
+| FloodControlField.js | used to enable a honeypot field to deter bots | not in use (?) |
+| FormActions.js | various methods to interact with forms | public and planning area |
+| GlobalEventListener | Add Event listeners, that have been added & initialized in mounted |
+| HighlightHashLink.js | used to highlight a currently active hashlink | used in faqlist.js (public) |
+| NotificationStoreAdapter.js | used to display flash messages | puplic and planning area |
+| Pager.js | used to support a twig macro pager | public and planning area |
+| Progress.js | used to display a progress bar | Map.vue |
+| Tabs.js | used to add tabbing behaviour | public area |
+| ToggleAnything.js | Adds toggle behavior to elements |
+| ToggleSideMenu.js | in/out. Saves state in sessionStorage |
```

Comment:
```suggestion
| ToggleSideMenu.js | in/out. Saves state in sessionStorage |
| Tooltips.js | Initializes tooltips on elements with `o-tooltip` Css class |
```

Reply:
Done in https://github.com/demos-europe/demosplan-core/pull/673/commits/4e39a3da124fcef91a4fc6e2098ceb7b6cc17a39